### PR TITLE
V5 ga 129 system db

### PR DESCRIPTION
**Pull request by jiggsaw85**
**Jira task**: [https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?label=Screen_Conversion&selectedIssue=V5GA-129](https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?label=Screen_Conversion&selectedIssue=V5GA-129)

### CODE REVIEW

File: src/assets/styles/main.scss [hunk 1]
```diff
@@ -544,6 +544,18 @@ $color-light: hsl(0, 0%, 96%);
...
+}
+
+.w-120 {
+  width: 120px;
+}
+
+.w-200 {
+  width: 200px;
+}
+
```
Comment (Aviv-Galmidi) [inline src/assets/styles/main.scss:558]: Please use `width-100px` , `width-120px` , and `width-200px` if possible

File: src/components/FormButton.vue [hunk 2]
```diff
@@ -252,4 +253,8 @@ withDefaults(defineProps<CustomButton>(), {
   margin-left: 27px;
 }
 
+.color-red {
```
Comment (Aviv-Galmidi) [inline src/components/FormButton.vue:256]: Not used anywhere, can be removed

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. Selected namespace / key are not selected by default
New
Screen Shot 2023-05-18 at 14 04 07
Old
Screen Shot 2023-05-18 at 14 06 15

2. When creating a new namespace (and I assume key) - it doesn't show up in the dropdown
Screen Shot 2023-05-18 at 14 09 55

3. Changing namespaces doesn't change the data
4. Changing keys doesn't change the data
5. Changing namespaces doesn't load the new namespace keys

3 and 4 and 5
Screen Shot 2023-05-18 at 14 10 28

6. Buttons need a bit wider gap between them
Screen Shot 2023-05-18 at 14 13 01

**Comment by SrdjanReblaze** [comment]
all comments fixed

**Comment by Aviv-Galmidi** [comment]
Numbers 2, 5, and 6 were not fixed yet

**Comment by SrdjanReblaze** [comment]
> Numbers 2, 5, and 6 were not fixed yet

agree, sorry i typed all pr fixed accidently here, trying to solve this issue with dropdown

**Comment by SrdjanReblaze** [comment]
everything fixed @Aviv-Galmidi

**Comment by Aviv-Galmidi** [comment]
Merged as part of `ux-upgrade-next`

**Pull request closed without merging**